src/lightning_app/api/http_methods.py

```diff
@@ -19,6 +21,17 @@ def _signature_proxy_function():
     pass
 
 
+@dataclass
+class FastApiMockRequest:
+    headers: Optional[str] = None
```

Comment:
We can already [access the headers](https://fastapi.tiangolo.com/tutorial/header-params/). What we need is the raw `body` attribute:
```
AttributeError: 'FastApiMockRequest' object has no attribute 'body'
ERROR: None
```

Reply:
I thought you would add what you needed ;) My bad.